### PR TITLE
First web page created on Aug 6, 1991

### LINKED ISSUE #1: Build the 1st web page ever created in the internet
## Task Description

- Create a web page that will look like below. Take the content from __Content__ section below. The page is http://info.cern.ch/hypertext/WWW/TheProject.html.
- Git commit the content with the message `First web page created on Aug 6, 1991`
- Sent the Pull Request (PR)
- Get PR reviewed and closed (NOTE: PR closed means task is done. No PR will be merged in this repo.)

NOTE: It is hosted on a _NeXT computer at the European Organization for Nuclear Research, CERN._

## Preview
Screenshot 2020-11-30 at 8 02 34 PM

## Content

```
World Wide Web
The WorldWideWeb (W3) is a wide-area hypermedia information retrieval initiative aiming to give universal access to a large universe of documents.
Everything there is online about W3 is linked directly or indirectly to this document, including an executive summary of the project, Mailing lists , Policy , November's W3 news , Frequently Asked Questions .

What's out there?
Pointers to the world's online information, subjects , W3 servers, etc.

Help
on the browser you are using

Software Products
A list of W3 project components and their current state. (e.g. Line Mode ,X11 Viola , NeXTStep , Servers , Tools , Mail robot , Library )

Technical
Details of protocols, formats, program internals etc

Bibliography
Paper documentation on W3 and references.

People
A list of some people involved in the project.

History
A summary of the history of the project.

How can I help ?
If you would like to support the web..

Getting code
Getting the code by anonymous FTP , etc.
```

### PR DESCRIPTION
**Pull request by gorkhalitechie**
@coolbrg Sir, Fixed #1.

### CODE REVIEW

File: milestone_1/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+<body>
+  <h1>World Wide Web</h1>
+  <p>
+    The WorldWideWeb (W3) is a wide-area <a href="http://info.cern.ch/hypertext/WWW/WhatIs.html"> hypermedia</a>
+    information retrieval initiative aiming to give universal access to a large universe of documents.
+  </p>
+  <p>
+    Everything there is online about W3 is linked directly or indirectly to this document, including an
+    <a href="http://info.cern.ch/hypertext/WWW/Summary.html">executive summary</a> of the project,
+    <a href="#">Mailing lists</a>, <a href="http://info.cern.ch/hypertext/WWW/Policy.html">Policy</a> ,
```
Comment (brgnepal) [inline milestone_1/index.html:17]: @gorkhalitechie , why empty `#`?

File: milestone_1/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      If you would like to support the web..
+    </dd>
+    <dt>
+      <a href="http://info.cern.ch/hypertext/README.html">Getting code</a>
+    </dt>
+    <dd>
+      Getting the code by
+      <a href="http://info.cern.ch/hypertext/WWW/LineMode/Defaults/Distribution.html">anonymous FTP</a>, etc.
+    </dd>
+  </dl>
```
Comment (brgnepal) [inline milestone_1/index.html:87]: Nice work!

**Review by brgnepal — CHANGES_REQUESTED** [review]
Please check the suggestions!

File: milestone_1/index.html (hunk 1 above)
Comment (gorkhalitechie) [reply milestone_1/index.html:17]: Sir, the href in the original web page doesn't has the link associated.

**Comment by brgnepal** [comment]
@gorkhalitechie The solution looks good. Go ahead with the next assignment. 👍

**Pull request closed without merging**